projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -958,9 +958,11 @@ public JSONObject listAnalyses(String containerName) {
 
          String[] questionList = new String[questions.length()];
 
+         @SuppressWarnings("unchecked")
```

Comment:
This is the wrong solution. You should simply declare it as Iterator<?> instead of Iterator. Specifically, the cast is pointless if this actually were an Iterator<String>, and also the suppression is a dangerous one that should only be used in rare cases.